local/local.go

```diff
@@ -722,7 +722,7 @@ func untar(r io.Reader, dest string) error {
 			if err != nil {
 				return err
 			}
-			if _, err := io.Copy(fh, tr); err != nil {
+			if _, err := io.CopyN(fh, tr); err != nil {
```

Comment:
I think that `io.CopyN` gets 3 variables: https://golang.org/pkg/io/#CopyN
You can read more about the "Potential DoS vulnerability via decompression bomb" we got from Gosec and how to solve it [here](https://stackoverflow.com/questions/67327323/g110-potential-dos-vulnerability-via-decompression-bomb-gosec).

Reply:
Done @yaelharel thanks